libs/wingc/src/jsify/snapshots/base_class_captures_inflight.snap

```diff
@@ -48,8 +49,9 @@ const $helpers = require("@winglang/sdk/lib/helpers");
 const $macros = require("@winglang/sdk/lib/macros");
 module.exports = function({ $Base }) {
   class Derived extends $Base {
-    constructor({  }) {
-      super({  });
+    constructor($args) {
+      const {  } = $args;
+      super($args);
     }
```

Comment:
Maybe we can omit this constructor if there is no state to lift

Reply:
Done